### DISCUSSION TITLE
jsPsychSurveyMultiChoice, "required" attribute

### DISCUSSION THREAD

**Discussion by jshinnn** [jspsych/jsPsych#2829] [Q&A] [answered]
Hi, is there a way to make a question required, but remove the asterisk `*`?

![Screen Shot 2022-10-18 at 8 35 24 PM](https://user-images.githubusercontent.com/72616591/196591670-f6f7ab9d-c0ac-4fb3-83cc-438590b8a499.png)

Provided a screenshot to show the difference between adding the `required: true` attribute to `jsPsychSurveyMultiChoice` pages, as opposed to adding `required: false`.

**Comment by ChristopheBossens** [comment, marked as answer]
I think the fastest way would be to just edit the `plugin-survey-multi-choice.js` directly. On line 140-142 you have the following:

```
if (question.required) {
   html += "<span class='required'>*</span>";
}
```

If you delete that it will not show the asterisk but still have the question required.

**Comment by jodeleeuw** [comment]
@ChristopheBossens solution will work. You can also add a CSS rule that hides the `.required` class for `<span>` elements:

```html
<head>
<style> span.required { display: none; } </style>
</head>
```
